auto_nag/scripts/reminder.py

```diff
@@ -0,0 +1,146 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+from dateutil import parser
+from libmozdata.bugzilla import BugzillaUser
+from libmozdata import utils as lmdutils
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+REMINDER_TYPES = [
+    "test",
```

Comment:
@tomrittervg @mozfreddyb if we do https://github.com/mozilla/relman-auto-nag/pull/1464#issuecomment-1152517053, then we don't need a reminder for tests. Wdyt?

Reply:
I don't think it will work on it's own.

I am confused about how we would accomplish the 'detect tests attached but not landed'.  We would have to specifically detect non-abandoned phabricator revisions, because it's very common for the fuzzing team to attach tests that never turn into landable patches.  

But even if that part isn't that hard (I do seem to remember there's a nag for "r+ but not landed...") - the sec-approval team still needs to manually put a date _somewhere_ that the bot will need to read to avoid reminding people too soon (or late).  When it's okay to land tests is typically 2-3 weeks into the release-after-the-next's cycle, but not always.  It can depend also on upstream disclosure.  

(Which reminded me to add a tag specifically for 'disclosure' to remind us when a bug is nearing public disclosure at e.g. a conference.)

Given that sec-approval will have to put a date somewhere anyway, it seems like it's less work all around to just use this functionality for landing tests.

Reply:
@tomrittervg to avoid bugs where people forget to set a reminder, what do you think of asking to land the tests if enough time has passed (and so we are certain we can land them)? E.g. after 6 months have passed.
Another option could be to wait two weeks, and then add a needinfo to ask to set a reminder.

Reply:
For sec-approval tests, it's on the sec-approval team (i.e. me and Dan) to add the whiteboard tag; not developers.  I do most sec-approvals so I'm not going to forget, so I'm not too worried about it =)